spec/models/form526_submission_remediation_spec.rb

```diff
@@ -15,6 +15,13 @@
   end
 
   describe 'validations' do
+    context 'remediation_type validation' do
+      it do
```

Comment:
Please add a message to the `it` example block

Reply:
this spec is of this sort, using shoulda matchers
```
  describe 'associations' do
    it { is_expected.to belong_to(:form526_submission) }
  end
```
shoulda matchers do not typically get text blocks, as it's redundant with the syntax, as you can see in a few other places in the file. Would you still like me to add it?

Reply:
I added it, for comparison. doesn't hurt anything to have it : )